content/rs/databases/active-active/create.md

```diff
@@ -91,6 +91,10 @@ Every instance of an Active-Active database can receive write operations, and al
 
         {{<image filename="images/rs/screenshots/databases/active-active-databases/create-db-add-participating-clusters.png" alt="Add cluster panel.">}}{{</image>}}
 
+        {{<note>}}
```

Comment:
Maybe try "RAM-only" and something like "RAM-flash mixed" to make this sentence easier to read.